README.md

```diff
@@ -361,7 +394,11 @@ steps:
       echo ARCHITECTURE=${{ steps.flutter-action.outputs.ARCHITECTURE }}
       echo PUB-CACHE-PATH=${{ steps.flutter-action.outputs.PUB-CACHE-PATH }}
       echo PUB-CACHE-KEY=${{ steps.flutter-action.outputs.PUB-CACHE-KEY }}
+      echo CACHE-HIT=${{ steps.flutter-action.outputs.CACHE-HIT }}
+      echo PUB-CACHE-HIT=${{ steps.flutter-action.outputs.PUB-CACHE-HIT }}
     shell: bash
 ```
-[Alif Rachmawadi]: https://github.com/subosito
-[Bartek Pacia]: https://github.com/bartekpacia
+
+[Alif Rachmawadi](https://github.com/subosito)
+
+[Bartek Pacia](https://github.com/bartekpacia)
```

Comment:
these lines did not show up in the [README](https://github.com/subosito/flutter-action/blob/395322a6cded4e9ed503aebd4cc1965625f8e59a/README.md), so i changed to links that do show up

Reply:
thank you!

Reply:
well, they were actually supposed to now show up. they are link reference definitions.

see https://github.github.com/gfm/#link-reference-definitions

Reply:
oops, my bad 😅. thanks for explaining!